.github/workflows/master.yml

```diff
@@ -46,15 +40,6 @@ jobs:
           bin/pull-subtrees master
           git push
           git push origin HEAD:staging --force
-  empty-job:
```

Comment:
This was an artifact from another build step that is no longer around, and is being pruned.